dask_expr/concat.py

```diff
@@ -84,8 +84,23 @@ def _simplify_down(self):
         )
 
     def _simplify_up(self, parent):
-        # TODO: implement passing projections through
-        return
+        if isinstance(parent, Projection):
+            columns = parent.columns
+            columns_frame = [
+                sorted(set(frame.columns).intersection(columns))
```

Comment:
Can you explain the reason for sorting? Is this related at all to how pandas will order the columns after `concat`?

Reply:
We have to compare somehow to frame.columns, which is easier when they are sorted but not strictly necessary if we compare both lists after sorting

Reply:
Testing the expression structure also gets harder if these can occur in any order

Reply:
I'd prefer keeping the sorted condition if there is no requirement from cudf that would break this. Using a set would make the order non-deterministic, which restricts our ability to test the expression structure. So this is just an internal thing, not pandas related.

Reply:
Okay, that seems fair to me.